integration/spark/src/main/scala/org/apache/carbondata/spark/util/CarbonScalaUtil.scala

```diff
@@ -771,4 +773,14 @@ object CarbonScalaUtil {
     (response, endTime)
   }
 
+  def copy(oldLoc: String, newLoc: String): Unit = {
```

Comment:
If used only in test files, can move it to TestUtil / TestClass

Reply:
Done